site_defaults.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I think we'd want the example site defaults to be something that's global to the site (I think trying to define points here will just confuse people) -- e.g., using it to specific metadata.system.location.site -- which should be the same for every device!  This is essentially extracting the relevant JSON from the (e.g.) AHU-1 metadata:

  "system": {
    "location": {
      "site": "ZZ-TRI-FECTA",